[2025-03-16T20:12:26.928+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: frequent_flyers.delete_data scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:12:26.934+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: frequent_flyers.delete_data scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:12:26.935+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-16T20:12:26.941+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): delete_data> on 2025-03-15 00:00:00+00:00
[2025-03-16T20:12:26.944+0000] {standard_task_runner.py:57} INFO - Started process 205 to run task
[2025-03-16T20:12:26.946+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'frequent_flyers', 'delete_data', 'scheduled__2025-03-15T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/frequent_flyers.py', '--cfg-path', '/tmp/tmp0rueuoh8']
[2025-03-16T20:12:26.948+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask delete_data
[2025-03-16T20:12:26.958+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-16T20:12:26.983+0000] {task_command.py:415} INFO - Running <TaskInstance: frequent_flyers.delete_data scheduled__2025-03-15T00:00:00+00:00 [running]> on host b70acf7f0e92
[2025-03-16T20:12:27.036+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='frequent_flyers' AIRFLOW_CTX_TASK_ID='delete_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-15T00:00:00+00:00'
[2025-03-16T20:12:27.036+0000] {sql.py:274} INFO - Executing: 
DELETE FROM presentation.frequent_flyers
WHERE created_at < NOW() - INTERVAL '1 day';
[2025-03-16T20:12:27.041+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:12:27.048+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:12:27.053+0000] {sql.py:418} INFO - Running statement: 
DELETE FROM presentation.frequent_flyers
WHERE created_at < NOW() - INTERVAL '1 day';, parameters: None
[2025-03-16T20:12:27.056+0000] {sql.py:427} INFO - Rows affected: 0
[2025-03-16T20:12:27.063+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=frequent_flyers, task_id=delete_data, execution_date=20250315T000000, start_date=20250316T201226, end_date=20250316T201227
[2025-03-16T20:12:27.085+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-16T20:12:27.102+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T20:25:15.083+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: frequent_flyers.delete_data scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:25:15.089+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: frequent_flyers.delete_data scheduled__2025-03-15T00:00:00+00:00 [queued]>
[2025-03-16T20:25:15.089+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-16T20:25:15.096+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): delete_data> on 2025-03-15 00:00:00+00:00
[2025-03-16T20:25:15.099+0000] {standard_task_runner.py:57} INFO - Started process 235 to run task
[2025-03-16T20:25:15.101+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'frequent_flyers', 'delete_data', 'scheduled__2025-03-15T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/frequent_flyers.py', '--cfg-path', '/tmp/tmpy_g0zm5j']
[2025-03-16T20:25:15.103+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask delete_data
[2025-03-16T20:25:15.113+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-16T20:25:15.137+0000] {task_command.py:415} INFO - Running <TaskInstance: frequent_flyers.delete_data scheduled__2025-03-15T00:00:00+00:00 [running]> on host 711ad4f92c48
[2025-03-16T20:25:15.188+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='frequent_flyers' AIRFLOW_CTX_TASK_ID='delete_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-15T00:00:00+00:00'
[2025-03-16T20:25:15.189+0000] {sql.py:274} INFO - Executing: 
DELETE FROM presentation.frequent_flyers
WHERE created_at < NOW() - INTERVAL '1 day';
[2025-03-16T20:25:15.196+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:25:15.203+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:25:15.208+0000] {sql.py:418} INFO - Running statement: 
DELETE FROM presentation.frequent_flyers
WHERE created_at < NOW() - INTERVAL '1 day';, parameters: None
[2025-03-16T20:25:15.211+0000] {sql.py:427} INFO - Rows affected: 0
[2025-03-16T20:25:15.218+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=frequent_flyers, task_id=delete_data, execution_date=20250315T000000, start_date=20250316T202515, end_date=20250316T202515
[2025-03-16T20:25:15.242+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-16T20:25:15.259+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
